endpoint.go

```diff
@@ -777,7 +766,7 @@ func (ep *endpoint) Delete(force bool) error {
 
 	ep.releaseAddress()
 
-	return nil
+	return n.getEpCnt().DecEndpointCnt()
```

Comment:
With the new flow, I don't think the caller should get a failure if the endpoint count decrement failed, because the driver endpoint deletion and IP deallocation were succesfull.

Given we have the endpoint count reconciliation logic at boot, shouldn't we only log a warning and return a nil error ?